src/braket/circuits/circuit.py

```diff
@@ -108,17 +106,18 @@ def register_subroutine(cls, func: SubroutineCallable) -> None:
             Instruction('operator': 'H', 'target': QubitSet(Qubit(1),))
         """
 
-        def method_from_subroutine(self, *args, **kwargs) -> SubroutineReturn:
+        def method_from_subroutine(self, *args, **kwargs) -> SubroutineReturn:  # noqa: ANN001
```

Comment:
same as for D417. I'm starting to question if we should be adding any new noqa-s

Reply:
Still here.